Java program to implement quick sort

advaith-unnikrishnan opened the pull request:
QuickSort is a Divide and Conquer algorithm. It picks an element as pivot and partitions the given array around the picked pivot. The key process in quickSort is partition(). Target of partitions is, given an array and an element x of array as pivot, put x at its correct position in sorted array and put all smaller elements (smaller than x) before x, and put all greater elements (greater than x) after x. All this should be done in linear time.

advaith-unnikrishnan committed:
Java program to implement quick sort

QuickSort is a Divide and Conquer algorithm. It picks an element as pivot and partitions the given array around the picked pivot. The key process in quickSort is partition(). Target of partitions is, given an array and an element x of array as pivot, put x at its correct position in sorted array and put all smaller elements (smaller than x) before x, and put all greater elements (greater than x) after x. All this should be done in linear time.

namanvats commented:
I cannot merge it until you make your program in a new folder named sorting algorithms and name the program as "quick_sort_java.java"I have already mentioned it.

advaith-unnikrishnan committed:
Delete quick_sort.java

advaith-unnikrishnan committed:
Java program to implement insertion sort

Insertion sort is a simple sorting algorithm that works the way we sort playing cards in our hands. Insertion sort is used when number of elements is small. It can also be useful when input array is almost sorted, only few elements are misplaced in complete big array.
